Kotlin/src/main/kotlin/parrot/AfricanParrot.kt

```diff
@@ -0,0 +1,8 @@
+package parrot
```

Comment:
I think you copied the Java version when this file was included there - these three subclasses were added in error and this should be removed since they are part of the solution